interp/src/environment.rs

```diff
@@ -1,18 +1,25 @@
 //! Environment for interpreter.
 
-//use super::{primitives, update};
-use calyx::{ir, ir::CloneName};
+use super::{primitives, primitives::Primitive, values::Value};
+use calyx::ir;
 use serde::Serialize;
-use std::collections::BTreeMap;
-use std::collections::HashMap;
-//use std::rc::Rc;
+use std::collections::{BTreeMap, HashMap};
 
 /// The environment to interpret a Calyx program.
 #[derive(Clone, Debug)]
 pub struct Environment {
     /// Stores values of context.
     /// Maps component names to a mapping from the component's cell names to their ports' values.
-    pub map: HashMap<ir::Id, HashMap<ir::Id, HashMap<ir::Id, u64>>>,
+    //pub map: HashMap<ir::Id, HashMap<ir::Id, HashMap<ir::Id, Value>>>,
```

Comment:
Commented code. Intentional?

Reply:
Vestigial to my knowledge. Removing and marking as resolved.